domains/harshd.json

```
@@ -0,0 +1,11 @@
+{
+    "description": "Portfolio",
+    "repo": "https://github.com/harsh-writes-code/harsh-writes-code.github.io",
+    "owner": {
+        "username": "harsh-writes-code",
+        "email": "happydhimanbarara@gmail.com"
+    },
+    "record": {
+        "CNAME": "happydhimanbarara@gmail.com.github.io"
```

this should be your github username not your email
```suggestion
        "CNAME": "harsh-writes-code.github.io"
```